src/machines/modelingMachine.ts

```diff
@@ -578,7 +580,8 @@ export const modelingMachine = setup({
     'has valid selection for deletion': () => false,
     'is-error-free': () => false,
     'no kcl errors': () => {
-      return !kclManager.hasErrors()
+      const hasErrors = kclManager.hasErrors()
+      return !hasErrors
```

Comment:
https://github.com/KittyCAD/modeling-app/pull/6910/files#r2089843309 and like reusing it here as well